code/modules/mob/living/carbon/human/life/handle_disabilities.dm

```diff
@@ -9,6 +9,11 @@
 		if((prob(1)) && (paralysis < 1))
 			seizure(10, 1000)
 
+	if(disabilities & ANEMIA)
+		var/blood_volume = round(vessel.get_reagent_amount(BLOOD))
+		if((prob(4)) && (blood_volume > BLOOD_VOLUME_SAFE))
+			vessel.remove_reagent(BLOOD, 10)
```

Comment:
two changes here:
1. make it `blood_volume >= BLOOD_VOLUME_SAFE+10` so that the player isn't constantly taking 1 tick of suffocation damage
2. elsewhere, you should prevent their blood regeneration from raising above BLOOD_VOLUME_SAFE so that the blood level isn't just constantly pogoing up and down

Reply:
done!